TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/transform/SigV4AuthTransformerFactoryTest.java

```diff
@@ -44,18 +44,19 @@ public void testSignatureProperlyApplied() throws Exception {
         var mockCredentialsProvider = new MockCredentialsProvider();
         DefaultHttpHeaders expectedRequestHeaders = new DefaultHttpHeaders();
         // netty's decompressor and aggregator remove some header values (& add others)
-        expectedRequestHeaders.add("host", "localhost");
+        expectedRequestHeaders.add("Host", "localhost");
```

Comment:
Can you point to a test (for the replayer) that shows that when host (& maybe other header values) have irregular casing, that the signature still works just fine?  I noticed a number of Map() projections by copying which could have destroyed the case insensitive semantics of StrictCaseInsensitiveHttpHeadersMap and ListKeyAdaptingCaseInsensitiveHeadersMap.

Reply:
Added to [SigV4AuthTransformerFactoryTest.java](https://github.com/opensearch-project/opensearch-migrations/pull/835/files#diff-5d7d4229d1ede018e9c0028fc658fc3b06c246d3da24520d82ce040bd5afd836)